libs/ui/src/reports/contest_results_table.tsx

```diff
@@ -12,7 +12,8 @@ import { format, getTallyReportCandidateRows } from '@votingworks/utils';
 import { throwIllegalValue, assert, Optional } from '@votingworks/basics';
 
 import { Text, NoWrap } from '../text';
-import { tableBorderColor } from '../table';
+
+const tableBorderColor = 'rgb(194, 200, 203)';
```

Comment:
I didn't want to mess with the print styles, so I just inlined the old border color here for now.

Reply:
I am envisioning that, with eventually moving toward a different print infrastructure, we'll totally partition print and UI styles.